Recruitment/JobSpecs/why_would_you_join_us.md

```diff
@@ -10,6 +10,6 @@ These are the things that drive us. Do they drive you?
 
 We are looking for people who want to help us change the way companies think and approach problems; people who want to work with teams and individuals who share a passion for solving complex problems; people who want to own the entire problem, whether it’s about technology, ways of working, teams, or customer experience.
 
-We are looking for people who want to be treated like adults; people who neither need, nor want, someone else to tell them what needs to be done, but instead who can look at what’s happening, what’s needed, and figure out how they fit into it; people who are comfortable finding their own place.
+We are looking for people who want to be treated like adults; people who neither need, nor want, someone else to tell them what needs to be done, but instead who can look at what’s happening, what’s needed, and figure out how they fit into it; people who are comfortable finding their own place; people who can be egoless about the work they produce.

```

Comment:
I don't think people are egoless, so it's a hard sell.
The term egoless was one that we couldn't reach agreement on when we had the Identity conversation a couple of weeks ago. We couldn't agree on what the word meant, and in what context we meant it.
That's why it was left out of what I wrote. If we find a way to say it, or reach some kind of agreement on it, let's add it, but for now, we don't seem ready to use the language.